### ISSUE TITLE
The performance is bad,that average_recall is 40%。 And I trained with 1080ti , at the same time ,I just change the batch-size to 8. Compared to yours paper's performance,why the result is so bad ? 

### ISSUE THREAD

**Issue by JXQI** [M3DV/FracNet#8]

**Comment by JXQI** [comment]
at the same time,I save the best model during the training,it's the 93 epoch,the val-loss is 0.4,and dice is 0.67,recall is 0.98

**Comment by kaimingkuang** [comment]
Hi @JXQI,
This issue is a duplicate of #7. 
Since we are also the host of the [RibFrarc Challenge](https://ribfrac.grand-challenge.org/), we didn't provide every detail in this repo to avoid data leakage of any kind. This repo is rather a baseline prototype that you can build your own model upon. There are a few things you can try to improve the performance:
- Larger batch size with multiple GPUs;
- Different training strategy, e.g., different scheduler;
- Better post-processing. We didn't reveal the entire postprocessing procedure since it involves our in-house code. This should be quite important.

Good luck!

**Comment by BelieferQAQ** [comment]
> at the same time,I save the best model during the training,it's the 93 epoch,the val-loss is 0.4,and dice is 0.67,recall is 0.98

Hello, I have encountered the same problem. Can I add your QQ and discuss it? Thank you very much. My qq318831418
